src/dpr/components/report-list/utils.ts

```diff
@@ -20,10 +21,15 @@ function getDefaultSortColumn(fields: components['schemas']['FieldDefinition'][]
   return defaultSortColumn ? defaultSortColumn.name : fields.find((f) => f.sortable).name
 }
 
+function isListWithWarnings(data: Dict<string>[] | ListWithWarnings): data is ListWithWarnings {
+  return (data as ListWithWarnings).data !== undefined
```

Comment:
should this be `.data !== undefined` or `.warnings !== undefined` ?

Reply:
Either would work - it just needs to be a property the other type doesn't have. :)